src/pluginManager.ts

```diff
@@ -285,8 +285,9 @@ export class PluginManager {
 
             const absolutePath = path.join(searchPath, scopeDirectory);
             fs.readdirSync(absolutePath)
+              .filter(name => PluginManager.isQualifiedPluginIdentifier(name))
               .filter(name => fs.statSync(path.resolve(absolutePath, name)).isDirectory())
-              .forEach(name => relativePluginPaths.push(path.join(scopeDirectory, name)));
+              .forEach(name => relativePluginPaths.push(scopeDirectory + "/" + name));
```

Comment:
By not using `path.join` and assembling the plugin by ourself and forcing a `/` we remove the need to have the replace thing in the PluginManager.isQualifiedPluginIdentifier function and also restore functionality for our activePlugins.includes check.